pkg/envoy/sds/response.go

```diff
@@ -85,17 +113,17 @@ func getEnvoySDSSecrets(cert certificate.Certificater, proxy *envoy.Proxy, reque
 			continue
 		}
 
-		if serviceForProxy != sdsCert.MeshService {
-			log.Error().Msgf("Proxy %s (service %s) requested service certificate %s; this is not allowed", proxy.GetCommonName(), serviceForProxy, sdsCert.MeshService)
+		if proxyService != sdsCert.MeshService {
+			log.Debug().Msgf("Proxy %s (service %s) requested service certificate %s; this is not allowed", s.proxy.GetCommonName(), proxyService, sdsCert.MeshService)
```

Comment:
`Debug` makes sense, except I worry that it would hide this message, when I believe it should be emitted with a higher level of urgency. If error seems to high - I'd say warn.

Reply:
`Warn` seems appropriate for this use case, will address!

Reply:
Addressed in https://github.com/openservicemesh/osm/pull/1995